packages/math/src/integers.ts

```diff
@@ -80,6 +105,8 @@ export class Uint32 implements Integer, WithByteConverters {
   }
 }
 
+const _uint32ClassConformsToStaticInterface: FixedLengthIntegerStatic<Uint32> = Uint32;
```

Comment:
Are these just to check the types? Maybe they make more sense in a test file?

Reply:
Yes, in eb63957b08d16ddb2209d08407d464e27beda670 I clarified that a bit. It is the static version of the `implements` keyword. It could also be written as

I think this check belongs to the definition. If we move this to a test we need to make the interfaces public, which I would like to avoid if possible.